src/cleaning_functions.py

```diff
@@ -24,26 +24,29 @@ def get_label_encoded_feature(feature):
     return [label_encoded_feature]
 
 
-def get_one_hot_encoded_feature(feature):
+def get_one_hot_encoded_feature(feature, feature_name):
```

Comment:
Umm.. Let's say we are encoding company names. How can I actually know if the feature is named 'company' if I dont take feature name as string ?